app/src/main/java/com/droidhats/campuscompass/views/MapFragment.kt

```diff
@@ -95,11 +102,8 @@ class MapFragment : Fragment(), OnMapReadyCallback, GoogleMap.OnMarkerClickListe
         super.onActivityCreated(savedInstanceState)
         viewModel = ViewModelProviders.of(this).get(MapViewModel::class.java)
 
-        if(activity != null)
```

Comment:
Is there a reason we are removing this check? What happens if the activity is equal to null at this point? Will it fail (I guess it would be unlikely)? Should we intentionally fail since running the app depends on running the activity?

Reply:
I thought it would be usless here. If the activity is null the app would have crashed already

Reply:
That's what I thought, thank you!